README.md

```diff
@@ -1,2 +1,189 @@
 # spotz [![Build Status](https://travis-ci.org/eHarmony/spotz.svg?branch=master)](https://travis-ci.org/eHarmony/spotz) [![Stories in Ready](https://badge.waffle.io/eHarmony/spotz.png?label=ready&title=Ready)](https://waffle.io/eHarmony/spotz) #
-Spark Parameter Optimization
+# Spark Parameter Optimization
+
+Spotz is a
+[hyperparameter optimization](https://en.wikipedia.org/wiki/Hyperparameter_optimization)
+framework written in [Scala](http://www.scala-lang.org) designed to exploit
+[Apache Spark](http://spark.apache.org) perform its distributed computation.
+A broad set of optimization algorithms have been implemented to solve for the
+hyperparameter values of an [objective function](https://en.wikipedia.org/wiki/Loss_function)
+that you specify.
+
+## Motivation
+The [eHarmony](http://www.eharmony.com) modeling team primarily uses Spark
+and Scala as the base of its machine learning pipeline.  Given that Spark
+is our distributed compute engine of choice, we have need for a robust
+hyperparameter optimization framework that integrates well with Spark.  There
+are already excellent frameworks out there like
+[Hyperopt](http://hyperopt.github.io/hyperopt) and
+[Optunity](http://optunity.readthedocs.io/en/latest),
+written in Python, but the ideal framework that runs in Scala on top of Spark
+does not exist.  [MLlib](http://spark.apache.org/mllib), though providing
+some support for Grid Search, is not a general framework for hyperparameter
+tuning and does not integrate with other learners.  This project's purpose is
+to build a simple framework that developers can integrate with Spark to fulfill
+their hyperparameter optimization needs.
+
+## VowpalWabbit
+At [eHarmony](http://www.eharmony.com), we make heavy use of
+[VowpalWabbit](https://github.com/JohnLangford/vowpal_wabbit/wiki).
+We use this learner so much that we feel strong integration with VW is very
+important.  Considering that VowpalWabbit does not support hyperparameter
+optimization out of the box, we've taken steps to support it without losing
+generality.
+
+## Optimization Algorithms
+
+Currently the following solvers have been implemented:
+
+* [Random Search](https://en.wikipedia.org/wiki/Random_search)
+* [Grid Search](https://en.wikipedia.org/wiki/Grid_search_method)
+* We are currently exploring other search algorithms to add
+
+## Maven dependency
+
+To use this as part of a maven build
+
+```xml
+<dependency>
+    <groupId>com.eharmony</groupId>
+    <artifactId>spotz-core</artifactId>
+    <version>1.0.0</version>
```

Comment:
I can't find the example you're referring to.  Did you mean in the pom file?  How do you set the version within the github readme markdown?